### DISCUSSION TITLE
/web/admin returning 404

### DISCUSSION THREAD

**Discussion by charlotattardjpj** [drakkan/sftpgo#455] [General]
Hi,

I have done the installation and connected the sftp with mysql database. But when im trying to access the web portal, to setup the amdin password im being returned to /web/users. 

Can you please help ?

Thanks,
Charlot

**Comment by drakkan** [comment]
Hi,

you are likely using v2.0.4 or older. The web admin setup screen was added after v2.0.4 and it will be included in v2.1.0.

In v2.0.4 you can login using `admin` as username and `password` as password. Alternately you can install the development version that will become v2.1.0 in few days

**Comment by charlotattardjpj** [comment]
Hi Nicola, 

Thanks a lot for your reply.

Yes I managed to install it now. im still having some issues when users trying to connect with it 

"Login failed. Exhausted available authentication methods. Please contact your web hosting service provider for assistance. Please contact your web hosting service provider for assistance." with this error.

Thankyou,
Charlot

**Reply by drakkan** [reply]
Hi,

this error message does not seem generated by SFTPGo, can you please post the full SFTPGo logs? Thank you

P.S. you can also try to follow the getting started guide [here](https://github.com/drakkan/sftpgo/blob/main/docs/howto/getting-started.md) but it refers to the development version

**Reply by charlotattardjpj** [reply]
Jun 16 11:56:11  sftpgo[109400]: {"level":"debug","time":"2021-06-16T11:56:11.852","sender":"sftpd","message":"failed to accept an incoming connection: ssh: disconnect, reason 2: too many authentication failures"}
Jun 16 11:56:11 ftpgo[109400]: {"level":"debug","time":"2021-06-16T11:56:11.852","sender":"connection_failed","client_ip":"172.16.114.108","username":"","login_type":"no_auth_tryed","protocol":"SSH","error":"ssh: disconnect, reason 2: too many authentication failures"}
Jun 16 11:56:40 sftpgo[109400]: {"level":"debug","time":"2021-06-16T11:56:40.932","sender":"connection_failed","client_ip":"172.16.114.108","username":"chaatt","login_type":"publickey","protocol":"SSH","error":"invalid credentials"}
Jun 16 11:56:44sftpgo[109400]: {"level":"debug","time":"2021-06-16T11:56:44.056","sender":"sftpd","connection_id":"cbf4b631a5bd8e71d61aba38081f778e7fa52a60946358f83f39079bfd08af6f","message":"cannot login user \"chaatt\", protocol SSH is not allowed"}
Jun 16 11:56:44 sftpgo[109400]: {"level":"debug","time":"2021-06-16T11:56:44.056","sender":"connection_failed","client_ip":"172.16.114.108","username":"chaatt","login_type":"password","protocol":"SSH","error":"Protocol SSH is not allowed for user \"chaatt\""}
Jun 16 11:56:47 sftpgo[109400]: {"level":"debug","time":"2021-06-16T11:56:47.774","sender":"sftpd","connection_id":"cbf4b631a5bd8e71d61aba38081f778e7fa52a60946358f83f39079bfd08af6f","message":"cannot login user \"chaatt\", protocol SSH is not allowed"}
Jun 16 11:56:47 sftpgo[109400]: {"level":"debug","time":"2021-06-16T11:56:47.774","sender":"connection_failed","client_ip":"172.16.114.108","username":"chaatt","login_type":"password","protocol":"SSH","error":"Protocol SSH is not allowed for user \"chaatt\""}
Jun 16 11:56:50  sftpgo[109400]: {"level":"debug","time":"2021-06-16T11:56:50.838","sender":"connection_failed","client_ip":"172.16.114.108","username":"chaatt","login_type":"password","protocol":"SSH","error":"invalid credentials"}
Jun 16 11:56:50 sftpgo[109400]: {"level":"debug","time":"2021-06-16T11:56:50.851","sender":"sftpd","message":"failed to accept an incoming connection: [ssh: no auth passed yet, Authentication error: could not validate public key credentials: invalid credentials, Authentication error: could not validate password credentials: Protocol SSH is not allowed for user \"chaatt\", Authentication error: could not validate password credentials: Protocol SSH is not allowed for user \"chaatt\", Authentication error: could not validate password credentials: invalid credentials]"}

**Reply by drakkan** [reply]
Hi,

the relevant error seems this one `Protocol SSH is not allowed for user "chaatt"`.

Please check the "Denied protocols" setting for the `chaatt` user, SSH should not be selected

![Schermata del 2021-06-16 14-09-54](https://user-images.githubusercontent.com/553263/122216376-a00f8300-ceac-11eb-95b8-d7bf7e2d16d4.png)
